### PR TITLE
feat: allow toggling multi-actions mode

### PR DESCRIPTION
**Pull request by fontanierh**
## Description

Kind of painful  to have to manually add `multi_actions=true` in the URL bar. The goal here is to increase internal adoption

Screenshot 2024-05-15 at 15 25 04

Screenshot 2024-05-15 at 15 25 12

### CODE REVIEW

File: front/pages/w/[wId]/builder/assistants/new.tsx [hunk 1]
```diff
@@ -65,10 +66,9 @@ export const getServerSideProps = withDefaultUserAuthRequirements<{
     };
   }
 
-  let multiActionsMode = context.query.multi_actions === "true";
-  if (multiActionsMode && !owner.flags.includes("multi_actions")) {
-    multiActionsMode = false;
-  }
+  const multiActionsAllowed = owner.flags.includes("multi_actions");
+  const multiActionsEnabled =
+    multiActionsAllowed && context.query.multi_actions === "true";
```
Comment (spolu) [inline front/pages/w/[wId]/builder/assistants/new.tsx:71]: can we kill the query stuff?

**Review by spolu — APPROVED** [review]
LGTM

File: front/pages/w/[wId]/builder/assistants/new.tsx (hunk 1 above)
Comment (fontanierh) [reply front/pages/w/[wId]/builder/assistants/new.tsx:71]: Yeah maybe we could/should

Comment (fontanierh) [reply front/pages/w/[wId]/builder/assistants/new.tsx]: done

**Pull request merged**